internal/workflow/processing.go

```diff
@@ -110,6 +111,13 @@ type TransferInfo struct {
 	// It is populated by the workflow request.
 	GlobalTaskQueue       string
 	PreservationTaskQueue string
+
+	// Send to failed variables used to keep track of the SIP
+	// location, if it requires zipping (a3m PIP) and what activity
+	// needs to be called to be uploaded to the expected bucket.
+	SendToFailedPath         string
+	SendToFailedActivityName string
+	SendToFailedNeedsZipping bool
```

Comment:
We have an `isDir` variable that I initially re-used for this. The issue was in preprocessing, where `isDir` is set to true even if it "fails". When preprocessing fails, we still want to upload the original so I ended with an ugly condition to avoid zipping in that case.

It's important to differentiate the current status of the package vs. the moment we want to use to upload to the failed bucket, that's why I added the new variables.